docs/samples/Create Asset.md

```diff
@@ -0,0 +1,295 @@
+## 1. Optional - Local Setup
+
+TODO link to a 0.4.x example setup.
+
+## 2. Terminology
+
+With the migration to a new protocol and with it new specifications, DCAT Catalogs in the context of assets, there is a need to clarify the terminology that is used.
+In the table below, relevant terms from the connector and their new counterparts are listed and explained with a short description.
+
+| Connector           | New Protocol (DCAT Catalogs) | Description                                                |
+|---------------------|------------------------------|------------------------------------------------------------|
+| Asset Entry         | Asset                        | Contains the Asset ID and its Data Address.                |
+| Contract Definition | Asset Entry / Dataset        | Contains an Asset that is offered and covered by a Policy. |
+
+Generally, this documentation uses the connector terminology unless otherwise specified.
+
+## 3. Values
+
+Since some keys which are required in requests for the new management API aren't self-explanatory when you first see them, a short explanation is given below.
+These keys are generally part of the JSON-LD spec and serve as identification on a larger scope.
+
+| Key      | Description                                                                                                                          |
+|----------|--------------------------------------------------------------------------------------------------------------------------------------|
+| @context | JSON-LD context aims to enable the local usage of compact and meaningful names which are later still identifiable in a larger scope. |
+| @vocab   | The value of `@vocab` is added as prefix to any key consisting of a plain string without any other prefix.                           |
+| edc      | `edc` defines a specific prefix for the connector.                                                                                   |
+| @id      | `@id` contains a distinct identifier enabling global references to this field, also called a Node.                                   |
+| @type    | Specifies the path to the schema which describes this Node.                                                                          |
+
+> Please note: The namespaces in `@vocab` and `edc` currently are only placeholders and do not lead to any JSON-LD context definition.
+> This will change at a later date.
+
+## 4. Simple Asset
+
+The first use-case is to generate a very simple asset, that only contains the minimum in terms of information.
+For this we need both an asset and a data address, which together form an asset entry.
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "edc:asset": {
+    "@type": "edc:asset",
+    "@id": "some-asset-id"
+  },
+  "edc:dataAddress": {
+    "@type": "edc:dataAddress",
+    "type": "test-type",
+    "keyName": "test-key-name"
```

Comment:
In my opinion  using a real type of dataAddress would be more beneficial. E.g. HttpData